pkg/apis/ceph.rook.io/v1/validate_resources_test.go

```diff
@@ -0,0 +1,88 @@
+package v1
+
+import (
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+func TestCephClusterValidateCreate(t *testing.T) {
+	c := &CephCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rook-ceph",
+			Namespace: "rook-ceph",
```

Comment:
You likely don't even need to set the Namespace for the test.

Reply:
removed. I've just kept the name and datadirhostpath